### PR TITLE
🤖 [ergoCubSN002] Add configuration files

### PR DESCRIPTION
**Pull request by GiulioRomualdi**

### CODE REVIEW

File: src/WalkingModule/app/robots/ergoCubSN002/dcm_walking_with_joypad.ini [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+skip_dcm_controller                  1
+
+# The goal port receives 2 or 3 doubles according to the selected planner controller
+goal_port_suffix                    /goal:i
+
+# Scales the input coming from the goal port
+goal_port_scaling                   (0.5, 1.0, 0.5)
+
+# How much in advance the planner should be called. The time is in seconds
+planner_advance_time_in_s           0.08
```
Comment (S-Dafarra) [inline src/WalkingModule/app/robots/ergoCubSN002/dcm_walking_with_joypad.ini:23]: This value is other than the others. Also the ``max_feedback_delay_in_s`` is missing. I guess we had this issue also in the other ergoCub

**Comment by S-Dafarra** [comment]
cc @carloscp3009

File: src/WalkingModule/app/robots/ergoCubSN002/dcm_walking_with_joypad.ini (hunk 1 above)
Comment (GiulioRomualdi) [reply src/WalkingModule/app/robots/ergoCubSN002/dcm_walking_with_joypad.ini]: Done here: https://github.com/robotology/walking-controllers/pull/175/commits/be143cb5f6880e35264a4e8877cd08927df4768d

**Comment by GiulioRomualdi** [comment]
Hi @S-Dafarra I modified the `ini` as you suggested in https://github.com/robotology/walking-controllers/pull/175#discussion_r1570132900

If it is fine I think we can merge it

**Review by S-Dafarra — APPROVED** [review]

**Pull request merged**